bignerdranch/Deferred
# Adds module map to explicitly use Atomics

#### What's in this pull request?

We noticed an issue when using `carthage_cache` to pull in a pre-built Deferred 3.0, where when building our app we'd see compiler errors like this:

`LivingObjectCounter.swift:10:8: Missing required module 'Atomics'`

I learned about module maps and considered that it might be necessary to `use Atomics` in the module map for Deferred. Since that module map was automatically generated, I grabbed the generated version (at least the important bits) and added `use Atomics` to it.

#### Testing

1. Create a project that uses Carthage to build Deferred (pointing at this branch)
1. Build Deferred locally using `carthage bootstrap` or what not.
1. Build the test app and see that it works
1. Use `carthage_cache publish` to put that built product up on S3
1. move away the local Carthage/Build directory
1. `carthage_cache install`
1. build the test app and see that it works
